pyhive/presto.py

```diff
@@ -148,7 +151,7 @@ def execute(self, operation, parameters=None):
         if self._session_props:
             headers['X-Presto-Session'] = ','.join(
                 '{}={}'.format(propname, propval)
-                for propname, propval in self._session_props.items()
+                for propname, propval in list(self._session_props.items())
```

Comment:
It isn't... I just didn't notice it after doing using the python 2 -> python 3 conversion tool. I removed such copies elsewhere. Thanks for spotting it!